modules/backing-up-red-hat-quay.adoc

```diff
@@ -148,39 +203,54 @@ $ export AWS_SECRET_ACCESS_KEY=$(oc get secret -l app=noobaa -n <quay-namespace>
 [source,terminal]
 ----
 $ mkdir blobs
-----
-+
-[source,terminal]
-+
-----
+
 $ aws s3 sync --no-verify-ssl --endpoint https://$(oc get route s3 -n openshift-storage  -o jsonpath='{.spec.host}')  s3://$(oc get cm -l app=noobaa -n <quay-namespace> -o jsonpath='{.items[0].data.BUCKET_NAME}') ./blobs
 ----
 
-
 [NOTE]
 ====
 You can also use link:https://rclone.org/[rclone] or link:https://s3tools.org/s3cmd[sc3md] instead of the AWS command line utility.
 ====
 
-. Scale up the Quay the Quay Operator:
+=== Scale the {productname} deployment back up
+
+. *For operator version 3.7 and newer*: scale up the {productname} deployment by re-enabling auto scaling (if desired) and removing the replica overrides for Quay, Mirror workers and Clair as appliable. Your `QuayRegistry` resource should look similar to the following:
 +
-[source,terminal]
-----
-$  oc scale --replicas=1 deployment $(oc get deployment -n <quay-operator-namespace> |awk '/^quay-operator/ {print $1}') -n <quay-operator-namespace>
+[source,yaml]
 ----
+apiVersion: quay.redhat.com/v1
+kind: QuayRegistry
+metadata:
+  name: registry
+  namespace: ns
+spec:
+  components:
+    …
+    - kind: horizontalpodautoscaler
+      managed: true <1>
+    - kind: quay <2>
+      managed: true 
+    - kind: clair
+      managed: true
+    - kind: mirror
+      managed: true
+    …
+----
+<1> re-enables auto scaling of Quay, Clair and Mirroring workers again (if desired)
+<2> replica overrides are removed again to scale the Quay components back up
 
-. Scale up the Quay namespace:
+. *For operator version 3.6 and earlier*: scale up the {productname} deployment by scaling up the {productname} operator again:
 +
 [source,terminal]
 ----
-$ oc scale --replicas=1 deployment $(oc get deployment -n <quay-namespace> -l quay-component=quay -o jsonpath='{.items[0].metadata.name}') -n <quay-namespace>
+$  oc scale --replicas=1 deployment $(oc get deployment -n <quay-operator-namespace> |awk '/^quay-operator/ {print $1}') -n <quay-operator-namespace>
 ----
 
-. Check the status of the Operator:
+. Check the status of the {productname} deployment:
 +
 [source,terminal]
 ----
-$ oc get quayregistry <quay-registry-name> -n <quay-namespace> -o yaml
+$ oc wait quayregistry registry--for=condition=Available=true
```

Comment:
I tried below steps in 3.4 , 3.5, 3.6 and 3.7
* deployment quay
* scale down 
* scale up
* oc wait quayregistry quayregistry --for=condition=Available=true  --timeout=360s -n quay-enterprise

I found it works as expected in 3.6 and 3.7.

In 3.4,  the `oc wait.... ` keep `met` result during quay pod reconciling. 
```
$ oc wait quayregistry quayregistry --for=condition=Available=true  --timeout=360s -n quay-enterprise34
quayregistry.quay.redhat.com/quayregistry condition met

$ oc get  quayregistry quayregistry  -n quay-enterprise34 -o yaml
apiVersion: quay.redhat.com/v1
kind: QuayRegistry
.......
status:
  conditions:
  - lastTransitionTime: "2022-06-22T05:46:37Z"
    lastUpdateTime: "2022-06-22T05:46:37Z"
    message: all registry component healthchecks passing
    reason: HealthChecksPassing
    status: "True"
    type: Available
  - lastTransitionTime: "2022-06-22T06:14:42Z"
    lastUpdateTime: "2022-06-22T06:14:42Z"
    message: all objects created/updated successfully
    reason: ComponentsCreationSuccess
    status: "False"
    type: RolloutBlocked
  configEditorCredentialsSecret: quayregistry-quay-config-editor-credentials-9c5kcdgfc9
  configEditorEndpoint: https://quayregistry-quay-config-editor-quay-enterprise34.router-default.apps.whu411az7.qe.azure.devcluster.openshift.com
.......
```

In 3.5, the `oc wait .....` will wait untill timeout. 
```
$  oc get quayregistry quayregistry -n quay-enterprise35 -o yaml
apiVersion: quay.redhat.com/v1
kind: QuayRegistry
metadata:
  ........
  uid: e6021322-90c5-46e4-975a-f8e835840754
spec:
  components:
  - kind: postgres
    managed: true
  - kind: objectstorage
    managed: true
  - kind: redis
    managed: true
  - kind: horizontalpodautoscaler
    managed: true
  - kind: route
    managed: true
  - kind: mirror
    managed: true
  - kind: monitoring
    managed: false
  - kind: clair
    managed: true
  configBundleSecret: config-bundle-secret
status:
  configEditorEndpoint: https://quayregistry-quay-config-editor-quay-enterprise35.apps.whu411az7.qe.azure.devcluster.openshift.com
  currentVersion: 3.5.7
  lastUpdated: 2022-06-22 07:09:10.448971301 +0000 UTC
  registryEndpoint: https://quayregistry-quay-quay-enterprise35.apps.whu411az7.qe.azure.devcluster.openshift.com

$ oc wait quayregistry quayregistry --for=condition=Available=true  --timeout=360s -n quay-enterprise34
hung
```
If we put 3.4 ans 3.5 in low priority,  we can use `oc wait......` here. If we need make it works in all supported quay version, it is better to suggest customer to check if all related pods are running.

Reply:
Yes, the status conditions where only added in 3.5 I think and the 3.6 refinements should make them reliable. I think we can ignore 3.5 since it will be EOL in a couple of months.